### PR TITLE
Portfolio website Desktop version

### PR DESCRIPTION
**Pull request by sylvester-wamaya**
- Add desktop styling version to the `nav` bar
- Add desktop styling version to the Home section.
- Add the desktop styling version to the works section.
- Add desktop styling version to the About me section.
- Add desktop styling version to the contact form section.
- Add desktop styling version to the `footer`.
- Add button interaction styling.

### CODE REVIEW

**Review by amon-cofie — CHANGES_REQUESTED** [review]
Hi @sylvester-wamaya , 👋 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 💪 



## Highlights

🚀 You have a descriptive pull request summary
🚀 You have a professional Readme file
🚀 You have good commit messages
🚀 You used GitHub flow properly
🚀 You have no linter errors

## Required Changes ♻️

- [ ] You have done awesome work on this project and your page looks almost like a clone of the figma template 👏 👍. However, this is your portfolio and in your works section you would want your user and hence your potential employer to be able to get a link to the project on GitHub 😉. I suggest in that light that you add the ```"see project"``` button and remove the dark background so that the project image can be seen when you hover on the projects in the works section. To put this in perspective, the first card in the figma template illustrates how the project card will look when it is hovered and the other cards illustrate how the projects will look on the page when not hovered. Check the screenshots below.

image

- [ ] Your desktop version looks really great and you should make sure your mobile version also looks as good 😉 Your navigation menu shows on your mobile page in a way it should not but I'm sure this is an easy fix for you 👍 💪 . Please check the screenshot below: 

Your design: 
image

Figma design: 
image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: mobile.css [hunk 1]
```diff
@@ -1,9 +1,7 @@
 
 /* Mobile responsiveness */
 
-html {
-  font-size: 62.5%;
-}
+
```
Comment (tresorsawasawa) [inline mobile.css]: - [ ] As mentioned by the previous reviewer and for the project requirements, kindly make sure your navigation menu options are not visible in a mobile screen. To handle that, you should use media queries in your CSS file to make your desktop version look like the template you selected (Desktop: >=768px). In screen size < 768px only the Hamburger shoulb visible and the one >= 768px then your navigation options should visible and the Hamburger hidden. For a short remind about media queries, just refer this **[article](https://www.w3schools.com/css/css_rwd_mediaqueries.asp)**. 
### Your Design in mobile size

mob

**Review by tresorsawasawa — CHANGES_REQUESTED** [review]
# STATUS: REQUEST CHANGES :recycle::recycle: 

Hi @sylvester-wamaya 
Great work on making the changes requested by the previous reviewer  👏👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me [@tresorsawasawa](https://github.com/tresorsawasawa) when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by nahnahdev — APPROVED** [review]
## Project Status: APPROVED :green_circle: 
Hi @sylvester-wamaya , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Descriptive pull request summary :heavy_check_mark: 
- Professional Readme file :heavy_check_mark: 
- Good commit messages :heavy_check_mark: 
- Good use of GitHub flow :heavy_check_mark: 
- No linter errors :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,704 @@
...
+    font-family: 'Crete Round', sans-serif;
+    font-size: 10.7vw;
+    font-weight: 400;
+  }
+
+  .line {
+    border: 0.3vw solid #212121;
+    width: 100%;
+    margin: 6.4vw 0 21.3vw 0;
+  }
```
Comment (MasterLeiva) [inline styles.css:160]: I suggest that the div element be styled using the background color attribute and giving a height to the div of 1px.

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,704 @@
...
+  .grey-button {
+    float: left;
+    background-color: rgba(255, 255, 255, 0.24);
+    color: #fff;
+    border: inherit;
+    font-weight: 500;
+    font-size: 1vw;
+    padding: 0.7vw 0.83vw;
+    margin-right: 0.83vw;
+  }
```
Comment (MasterLeiva) [inline styles.css:547]: I kindly suggest you turn the container of the buttons into a flex container so that you don't have hover issues.

**Review by MasterLeiva — COMMENTED** [review]
Great job on your portfolio design so far!
We kindly suggest you take a look into the following issues so that you can improve your website! 🙌🏼

_Take a look at the comments below_

**Pull request merged**